README.md

```diff
@@ -0,0 +1,115 @@
+
+## 🛠 Tech-stack <a name="tech-stack"></a>
+
+This project is implemented using the Ruby on Rails programming language, known for its elegant syntax and flexibility. RoR provides a suitable foundation for this Blog_App code decoding project due to its ease of use and string manipulation capabilities.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+This project is built using Ruby on Rails, a dynamic, open-source programming language known for its simplicity and productivity. Ruby's elegant syntax and powerful features make it a popular choice among developers.
+
+## Key Features <a name="key-features"></a>
+- [x] **Rails new (setup)**
+- [x] **Data Model**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get started with the Catalog of My Things app, follow the instructions below:
+
+### Setup <a name="setup"></a>
+
+1. Clone the project repository:
+   bash
+   `git clone https://github.com/ClaudiaRojasSoto/Blog_App.git`
+   
+
+2. Navigate to the project folder:
+   bash
+   `cd Blog_App`
+
+   
+
+### Prerequisites <a name="prerequisites"></a>
+
+To run the Catalog of My Things app, you'll need the following prerequisites:
+
+- An integrated development environment (IDE) like Visual Studio Code.
+- Ruby installed on your machine.
+- Git installed on your machine.
+- A GitHub account for version control.
+
+## Usage <a name="usage"></a>
+
+Follow these steps to use the program:
+
+1. Open your terminal.
+2. Navigate to the directory where the program files are located using the `cd` command:
+
+   bash
+   `cd /path/to/your/program/files`
+   
+
+3. Start your Ruby on Rails server
+
+   bash
+   `rails s`
+
+
+4. cess the Blog app in your web browser at `http://localhost:3000`.
+
+5. You can now interact with the app, create posts, add comments, and explore its features.
+
+
+## 👥 Authors <a name="authors"></a>
+
+This project was developed by:
+
+👤 **Claudia Rojas**
+-   GitHub: [@ClaudiaRojas](https://github.com/ClaudiaRojasSoto)
+-   LinkedIn: [@ClaudiaRojas](https://www.linkedin.com/in/claudia-rojas-soto/)
+
+## ❓ FAQ <a name="faq"></a>
+
+*Q: Is there a plan to add more features to this application?*
+A: Yes, the author has plans to enhance this application's capabilities. One upcoming feature is to make the application executable via tactil, providing more convenience and accessibility to users.
+ 
+
+*Q: Where can I find more information about the Ruby programming language?*
+A: For more information about Ruby, its features, and its community, visit the official [Ruby website](https://www.ruby-lang.org/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! Feel free to check the [issues page](https://github.com/ClaudiaRojasSoto/Catalog_Of_My_Things/issues) and contribute to the project.
```

Comment:
- [optional] I think it would be nice if you put the correct link to the issue, this link is for the catalog of my things project and not the blog app, kindly fix that 🙏🏾

Reply:
Oh, Thank you @cynthiainga. I'll fix that!